graf3d/eve7/inc/ROOT/REveElement.hxx

```diff
@@ -16,10 +16,21 @@
 #include <ROOT/REveVector.hxx>
 #include <ROOT/REveProjectionBases.hxx>
 
-#include <map>
 #include <memory>
+#include <list>
+#include <ostream>
+#include <set>
+#include <string>
+
+#if __has_include(<nlohmann/json_fwd.hpp>)
+#include <nlohmann/json_fwd.hpp>
+#else
+// This forward declaration works for versions before 3.11, which introduced
+// "versioned, ABI-tagged inline namespace[s]". Fortunately, we can assume the
+// json_fwd.hpp header to be present starting from that same version.
```

Comment:
```suggestion
// This forward declaration works for versions before 3.11.0, which introduced
// a "versioned, ABI-tagged inline namespace". The json_fwd.hpp header is
// present in multi-header installations of version 3.11.0+, and single-header
// installations of version 3.11.2+.
```

Reply:
I've slightly changed the text as `json_fwd.hpp` was already present in multi-header installations pre-3.11.0, and we try to use it if available.